syntax = "proto2";

package object_detection.protos;

import "object_detection/protos/faster_rcnn.proto";
import "object_detection/protos/ssd.proto";

// Top level configuration for DetectionModels.
message DetectionModel {
  oneof model {
    FasterRcnn faster_rcnn = 1;
    Ssd ssd = 2;
  }

  // Path to file which has initial values of Variables.
  // It is similar to train.fine_tune_checkpoint, but not ckpt file.
  optional string init_file = 4 [default=""];
}
